Sources/App/Views/DocumentationPageProcessor.swift

```diff
@@ -22,16 +22,22 @@ struct DocumentationPageProcessor {
     let repositoryOwnerName: String
     let repositoryName: String
     let packageName: String
+    let reference: String
+    let targets: [String]
 
     init?(repositoryOwner: String,
           repositoryOwnerName: String,
           repositoryName: String,
           packageName: String,
+          targets: [String],
+          reference: String,
```

Comment:
nit: reverse order to other two additions
```suggestion
          reference: String,
          targets: [String],
```